pilot5-submission/pilot5-output/pilot5-tlfs/rtf/tlf-primary-pilot5.rtf

```diff
@@ -52,14 +52,14 @@
 }
```

Comment:
There is a number of changes here. The bot didn't catch them but I wonder if these are an issue?